DAQ_System/Source/main.cpp

```diff
@@ -1,10 +1,116 @@
 #include "mbed.h"
+#include <cstdio>
+#include <cstdlib>
 
-// main() runs in its own thread in the OS
-int main()
-{
-    while (true) {
 
+#include "L3GD20H.h"
+// Found in: https://os.mbed.com/users/julioefajardo/code/L3GD20H/docs/tip/classL3GD20H.html
+
+
+#define sensitivity (.00875 )// this is using a sensitivity of 250 dps which is equivalent to .00875
+#define DPS_TO_RadS (3.14159/180) // this is the conversion factor for degrees per second to radians per second all you have to do is multiply the callibrated data or the raw data by the conversion factor
```

Comment:
For 
constexpr float kDegreesToRadians = (3.14159/180);             line 11

it is not letting me put the 'f' after (3.14159/180) or inside the parenthesis like (3.14159/180)f or (3.14159/180f)
but it does let me put (3.14159f/180).

Reply:
Looks like you figured it out! 🎊 `f` works by the 3.14 because that's the floating-point value. `f` is to be used at initialization. In other words, by the time 3.14 is divided by 180, by default it is initialized into a double, not a float. In which case, casting would be necessary — but we are trying to avoid that.